### PR TITLE
Remove afnetworking

### PR DESCRIPTION
**Pull request by hhanesand**
Pretty straight forward removal of AFNetworking - now using NSURLSession instead

### CODE REVIEW

**Comment by hhanesand** [comment]
Ready for review! 👍  @heumn

File: Pod/Classes/ABEGithubAPIClient.m [hunk 1]
```diff
@@ -27,45 +26,75 @@ + (instancetype)sharedClient {
...
         }
-    }];
+        
+        success();
+    }] resume];
+}
+
+- (NSURLRequest *)saveIssueRequestForIssue:(ABEIssue *)issue errorHandler:(void (^)(NSError *error))errorHandler {
+    NSMutableURLRequest *baseIssueSaveRequest = [[self baseSaveIssueURLRequest] mutableCopy];
+    NSAssert([NSJSONSerialization isValidJSONObject:[issue toDictionary]], @"JSON Post body generated from issue is not valid JSON.");
```
Comment (heumn) [inline Pod/Classes/ABEGithubAPIClient.m:59]: Perfect!

File: Pod/Classes/ABEImgurAPIClient.m [hunk 2]
```diff
@@ -31,43 +34,94 @@ + (BOOL)isAPIKeySet {
...
+        
+        if (![NSURL URLWithString:urlString]) {
+            NSLog(@"There was an error getting the url of the uploaded image from the response body of the Imgur API.");
+            NSLog(@"Returned URL : %@", urlString);
+            errorHandler(nil);
+            return;
         }
-    }];
+        
+        success(urlString);
```
Comment (heumn) [inline Pod/Classes/ABEImgurAPIClient.m:79]: We should probably check if the success handler has been set? :)

File: Example/Podfile.lock [hunk 3]
```diff
@@ -1,21 +1,5 @@
...
-    - AFNetworking/Reachability
-    - AFNetworking/Security
-    - AFNetworking/Serialization
-  - AFNetworking/Reachability (3.0.4)
-  - AFNetworking/Security (3.0.4)
-  - AFNetworking/Serialization (3.0.4)
-  - AFNetworking/UIKit (3.0.4):
-    - AFNetworking/NSURLSession
-  - IssueReporter (1.0.0):
-    - AFNetworking
```
Comment (heumn) [inline Example/Podfile.lock:18]: Alt Text

File: IssueReporter.podspec [hunk 4]
```diff
@@ -20,5 +20,4 @@ Pod::Spec.new do |s|
   }
 
   s.public_header_files = 'Pod/Classes/**/*.h'
-  s.dependency 'AFNetworking'
```
Comment (heumn) [inline IssueReporter.podspec:23]: :fire:

**Comment by heumn** [comment]
Perfect.

File: Pod/Classes/ABEImgurAPIClient.m (hunk 2 above)
Comment (heumn) [reply Pod/Classes/ABEImgurAPIClient.m:79]: I merged but forgot about this one :) Should be fixed!

Comment (hhanesand) [reply Pod/Classes/ABEImgurAPIClient.m:79]: Will look at!

**Pull request merged**